docker/rust-builder/Dockerfile

```diff
@@ -1,20 +1,18 @@
-FROM rust:1.65.0-slim as base
+FROM ubuntu:22.04 as base
 WORKDIR /app
 
-RUN apt update; apt upgrade -y && \
+RUN apt-get update; apt-get upgrade -y && \
   apt install -y \
   build-essential \
   curl \
   g++-aarch64-linux-gnu \
   g++-x86-64-linux-gnu \
   libc6-dev-arm64-cross \
-  libclang-dev
+  libclang-dev \
```

Comment:
Wondering do we need to pin these versions?

Reply:
I wouldn't pin the package versions at this point (in my experience, nobody (including me) cared enough to keep that tidy in a long run), I would rather create a new lifecycle for the builder/runtime images (i.e. moving them to a separate repository and CI), and occasionally tag and snapshot those images.